test/k8sT/Conformance.go

```diff
@@ -76,5 +79,20 @@ var _ = Describe("K8sConformance", func() {
 			err := kubectl.WaitforPods(helpers.DefaultNamespace, "", helpers.HelperTimeout)
 			ExpectWithOffset(1, err).Should(BeNil(), "connectivity-check pods are not ready after timeout")
 		})
+
+		It("Check one node connectivity-check compliance with portmap chaining", func() {
+			SkipIfIntegration(helpers.CIIntegrationFlannel)
+			SkipItIfNoKubeProxy()
+
+			deployCilium(map[string]string{
+				"global.cni.chainingMode": "portmap",
+			})
+
+			kubectl.ApplyDefault(connectivityCheckYamlSimple).ExpectSuccess("cannot install connectivity-check-simple")
+
+			err := kubectl.WaitforPods(helpers.DefaultNamespace, "", helpers.HelperTimeout)
+			ExpectWithOffset(1, err).Should(BeNil(), "connectivity-check pods are not ready after timeout")
+		})
```

Comment:
If I understand correctly, we could use the existing test above if we were able to deploy several nodes in GitHub Actions? I think @sayboras was working on this (cf. #11738).

Reply:
Unfortunately this wouldn't really work for multi-node clusters, we would be running several Cilium agent instances on one kernel which would most probably wreak havoc in bpf filesystem and break everything, as we don't do any namespacing afaik (I don't know if this would even be possible).

Reply:
@pchaigno I saw this awesome PR, and thinking about trying to multi node in kind. It's for my own understanding only. There is one related PR https://github.com/cilium/cilium/pull/11715, which makes the connectivity check simpler as well.  As @nebril mentioned, might need to relax some condition/scope to make it working with multiple nodes in kind :)

Reply:
I don't think that's an issue. We even have a [GSG for a multi-node Kind setup](https://docs.cilium.io/en/latest/gettingstarted/kind/).

Reply:
Yeah, I followed the above _guide_ to setup my dev environment. My original plan is to use _chart testing_ with custom values coming from _ci_ directory (with the same value mentioned in guide). However, understanding from the team that having extra _ci_ folder might not be idea, so might need to wait for https://github.com/helm/chart-testing/issues/171 to be resolved first. @nebril is very kind to spend time explaining and pointing me to https://github.com/helm/chart-testing/issues/171 :tada:

Reply:
I totally missed this gsg guilde, thanks for this info!

Reply:
@nebril @pchaigno in that case, what do you think about this https://github.com/sayboras/cilium/pull/71/checks?check_run_id=720440156 ? FYI. I just copied details from this PR and GSG link?